uber/site_sections/registration.py

```diff
@@ -32,7 +32,7 @@ def checking_at_the_door(self, *args, **kwargs):
 
 @all_renderable(c.PEOPLE, c.REG_AT_CON)
 class Root:
-    def index(self, session, message='', page='0', search_text='', uploaded_id='', order='last_first', invalid=''):
+    def index(self, session, message='', page='1', search_text='', uploaded_id='', order='last_first', invalid=''):
```

Comment:
Agree. This change was made because there were huge performance issues with the whole list loading on every page. Long term there are better solutions, but until we totally rewrite how we render lists - Eli's suggestion gets a +1 from me.

> On Jan 19, 2016, at 12:01 PM, Eli Courtwright <notifications@github.com> wrote:
> 
> In uber/site_sections/registration.py:
> 
> > @@ -32,7 +32,7 @@ def checking_at_the_door(self, *args, **kwargs):
> >  
> >  @all_renderable(c.PEOPLE, c.REG_AT_CON)
> >  class Root:
> > -    def index(self, session, message='', page='0', search_text='', uploaded_id='', order='last_first', invalid=''):
> > +    def index(self, session, message='', page='1', search_text='', uploaded_id='', order='last_first', invalid=''):
> We definitely do NOT want this change for performance reasons, at least until such time as we optimize this page such that 50 different reg stations all loading this page constantly as they check people in and whatnot is fine.
> 
> What we DO want is for the first page to show up by default on search results. In other words, we should have page="0" here and then have an if-statement that if there's search text and the page is 0 then change the default page to 1.
> 
> —
> Reply to this email directly or view it on GitHub.
>